llvm/lib/Target/SystemZ/SystemZAsmPrinter.cpp

```diff
@@ -946,6 +950,21 @@ bool SystemZAsmPrinter::PrintAsmMemoryOperand(const MachineInstr *MI,
                                               unsigned OpNo,
                                               const char *ExtraCode,
                                               raw_ostream &OS) {
+  if (ExtraCode && ExtraCode[0] && !ExtraCode[1]) {
+    switch (ExtraCode[0]) {
+    case 'A':
+      // Unlike EmitMachineNode(), EmitSpecialNode(INLINEASM) does not call
+      // setMemRefs(), so MI->memoperands() is empty and the alignment
+      // information is not available.
```

Comment:
Hmm.  This is unfortunate, as it will lead to less efficient code being emitted without any indication to the user ...

I guess this is OK for now (probably still better than an error) - but can we look into actually fixing the underlying problem as well?

Reply:
Ok, I'll look into this. Ideally this will be as simple as adding an extra call, but, of course, there may be some fallout that will have to be investigated.